app/frontend/components/Stepper/Step.scss

```diff
@@ -64,3 +64,23 @@
   }
 }
 
+.submission-interstitial {
+  box-sizing: border-box;
+  height: 100vh;
```

Comment:
@eyko I need to import a mixin from the existing scss cause viewport units are broken on iPhone 5. Do you know how I can import the mixins file with webpack?